### PR TITLE
feature landingpage closes #141

### LINKED ISSUE #141: Style landingpage to match mockup
Currently the landing page is mostly a place holder
- [x] Create page accoridng to mockup: https://projects.invisionapp.com/share/J353G8DCW#/screens/118594700 
  - [x] Create html template in accordance with mockup. 
  - [x] Create styles to match mockup.
- [ ] Develop content @Shahin89 
  There is possible reuseable elements and style at Refugeetech.com

### PR DESCRIPTION
**Pull request by leosa83**
It still needs mobile adaptation with col-\* bootstrap classes
Other issues to be discovered

checkout #141

### CODE REVIEW

**Comment by leosa83** [comment]
please review @brylie

File: landing/client/style/landing.css [hunk 1]
```diff
@@ -5,15 +5,176 @@
...
+
+/* General Button Stuff */
+.btn:focus, .btn:active:focus, .btn.active:focus {
+    outline: 0 none;
+    background-color: #20245C;
+    color:#D92D6F;
+}
+
+ .btn:hover {
+  background-color: #20245C;
```
Comment (brylie) [inline landing/client/style/landing.css:160]: Inconsistent whitespace.

**Comment by brylie** [comment]
The HTML classes look good (Bootstrap grid classes). The files contain inconsistent whitespace, sometimes 1, 2, 3, or 4 spaces - clean up whitespace. Lastly, convert _px_ to _em_ values for layout and typography related CSS rules.

**Comment by brylie** [comment]
Resolve conflicts by calling _git pull origin develop_ from current working branch.

**Comment by brylie** [comment]
Try to make more frequent, and smaller, commits. Looking at the commit history for this PR does not give a clear indication of what changes or additions were made. Use a GUI to make granular commit workflow as simple as point, click, describe, commit.

**Comment by brylie** [comment]
By way of example, the three commits in this PR modify 2, 8, and 5 files respectively. Ideally, each file would receive its own commit summarizing significant changes. So, there would be something around 20 commits in a PR this size.

**Pull request merged**